src/core/store/modules/notifications/notifications.actions.ts

```diff
@@ -0,0 +1,37 @@
+import { createAction, createAsyncThunk } from '@reduxjs/toolkit';
+import { ThunkAPI } from '@frameworks/redux';
+
+import { Message } from '@types';
+import notificationMessages from '../../../../client/data/notificationMessages.json';
+
+const dismissMessage = createAction<number>('notifications/dismissMessage');
+
+/* -------------------------------------------------------------------------- */
+/*                                 Clear State                                */
+/* -------------------------------------------------------------------------- */
+
+const clearMessages = createAction<void>('notifications/clearMessages');
+const clearDismissedMessageIds = createAction<void>('notifications/clearDismissedMessageIds');
```

Comment:
Is this actually ever going to be used? Doesn't seem like it would be a user-facing option and I cannot see a case where we would actually like to clear the user dismissed messages

Reply:
Yeah, probably not needed as of now, so lets delete them so we can maintain only used code.